jest.config.js

```diff
@@ -15,7 +15,8 @@ module.exports = {
     // Force module uuid to resolve with the CJS entry point, because Jest does not support package.json.exports. See https://github.com/uuidjs/uuid/issues/451
     uuid: require.resolve('uuid'),
     '^nanoid(/(.*)|$)': 'nanoid$1',
-    'react-leaflet': '<rootDir>/static/src/js/util/mocks/reactLeafletMock.js'
+    'react-leaflet': '<rootDir>/static/src/js/util/mocks/reactLeafletMock.js',
+    '^axios$': require.resolve('axios')
```

Comment:
A comment would be useful here similar to the one for `uuid` I think, if I'm understanding correctly when we update to the latest `jest` we should be able to remove this resolution

Reply:
I can add a note.

Reply:
@eudoroolivares2016 I added the note! :)